instrumentation/apache-httpclient/apache-httpclient-5.2/library/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+  id("otel.library-instrumentation")
+  id("otel.nullaway-conventions")
+  id("otel.animalsniffer-conventions")
+}
+
+dependencies {
+  library("org.apache.httpcomponents.client5:httpclient5:5.2.1")
+  library("jakarta.annotation:jakarta.annotation-api:2.1.1")
```

Comment:
@trask Do we want to use javax here? The [recomendation](https://blogs.oracle.com/javamagazine/post/transition-from-java-ee-to-jakarta-ee) is to migrate to jakarta. Apache http client 5 uses jakarta, as well.

Reply:
I think it doesn't really matter which you use. It is just that you can use `javax.annotation.Nullable` without adding extra dependencies because of https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/8885012bcb6fd8da16291fd2edb3f5cc6bb0cfcb/conventions/src/main/kotlin/otel.java-conventions.gradle.kts#L139 I guess we could add the jakarta annotations there and here https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/8885012bcb6fd8da16291fd2edb3f5cc6bb0cfcb/dependencyManagement/build.gradle.kts#L104 as well

Reply:
Since the other modules already use javax, I changed to javax as well. When there is a need to upgrade to jakarta, it will be easier if we only have javax present.